app/views/notes/index.html.erb

```diff
@@ -6,6 +6,20 @@
            :commented => tag.span(t(".subheading_commented"), :class => "px-2 py-1 bg-body") %></p>
 <% end %>
 
+<%= form_with :url => url_for(:controller => "notes", :action => "index"), :method => :get, :data => { :turbo => true } do %>
```

Comment:
Why `:url => url_for(:controller => ...)` and not `:url => user_notes_path(@user)`?

Reply:
Missed this one. Refactored. Thank you.